<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="glCent">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbSetGeneral">
      <property name="title">
       <string>FFT / dBFS 분석 설정</string>
      </property>
      <layout class="QGridLayout" name="glSetGeneral">
       <item row="0" column="0">
        <widget class="QLabel" name="lbSize">
         <property name="text">
          <string>읽을 크기</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lbThread">
         <property name="text">
          <string>쓰레드 수</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="spSize">
         <property name="suffix">
          <string> KiB</string>
         </property>
         <property name="maximum">
          <number>524288</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="teStart">
         <property name="maximumTime">
          <time>
           <hour>0</hour>
           <minute>59</minute>
           <second>58</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QTimeEdit" name="teDuration">
         <property name="maximumTime">
          <time>
           <hour>0</hour>
           <minute>59</minute>
           <second>59</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spThread">
         <property name="suffix">
          <string> 개</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QCheckBox" name="chkEnableCut">
         <property name="text">
          <string>시작 시간/분석 길이 지정</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="gbSetThreshold">
      <property name="title">
       <string>임계값</string>
      </property>
      <layout class="QGridLayout" name="glSetThreashold">
       <item row="0" column="0">
        <widget class="QLabel" name="lbHDivM">
         <property name="text">
          <string>고/중</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbNDivM">
         <property name="text">
          <string>노이즈/중</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbdB">
         <property name="text">
          <string>dBFS 목표</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lbdBDiff">
         <property name="text">
          <string>dBFS 오차</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="QPushButton" name="btnApplyThreshold">
         <property name="text">
          <string>적용</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="spHDivM">
         <property name="suffix">
          <string> %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spNdivM">
         <property name="suffix">
          <string> %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spdB">
         <property name="suffix">
          <string> dBFS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spdBDiff">
         <property name="suffix">
          <string> dBFS</string>
         </property>
         <property name="prefix">
          <string>±</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="gbResult">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="glResult">
       <item row="0" column="0">
        <widget class="QPushButton" name="chkLoadRes">
         <property name="text">
          <string>결과 불러오기</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkIncludeFFTRes">
         <property name="text">
          <string>FFT 결과 포함</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="chkSaveRes">
         <property name="text">
          <string>결과 저장</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3" colspan="2">
     <widget class="QGroupBox" name="gbSetFreq">
      <property name="title">
       <string>주파수 대역 설정</string>
      </property>
      <layout class="QGridLayout" name="glSetFreq">
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="lbFreqL">
         <property name="text">
          <string>저</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="QLabel" name="lbFreqM">
         <property name="text">
          <string>중</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lbFreqML">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QSpinBox" name="spFreqML">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>24000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="6" rowspan="2">
        <widget class="QLabel" name="lbFreqH">
         <property name="text">
          <string>고</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="lbFreqHL">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QSpinBox" name="spFreqHL">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>24000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="9" rowspan="2">
        <widget class="QLabel" name="lbFreqN">
         <property name="text">
          <string>노이즈</string>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QLabel" name="lbFreqNL">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QSpinBox" name="spFreqNL">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>24000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbFreqLH">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spFreqLH">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>24000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="lbFreqMH">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QSpinBox" name="spFreqMH">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>23998</number>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="lbFreqHH">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QSpinBox" name="spFreqHH">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>23999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnAddFiles">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>파일 추가</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnStartAnalyse">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>분석 시작</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QTreeView" name="tvResult"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
